app/models/Departure.scala

```diff
@@ -44,6 +46,7 @@ trait BaseDeparture {
 
 case class Departure(
   departureId: DepartureId,
+  gbDepartureOffice: Option[Boolean],
```

Comment:
Could we use the `DepartureOffice` here instead? If we are going to save this to the database, I'd prefer that we save part of the domain model rather than a flag.

Reply:
Yes that's fine, but we will need to convert it to a Boolean for the response to our service each time we make a request (which could end up being every 5s).